templates/CWP/CWP/PageTypes/Layout/EventPage.ss

```diff
@@ -1,16 +1,18 @@
 <article class="container">
     <div class="row">
-        <section class="col-lg-8<% if not $Terms && not $RelatedPages %> offset-lg-2<% end_if %>">
-            <header class="page-header">
+        <section class="col-lg-12">
+            <div class="page-header">
```

Comment:
What was the reasoning around changing `header` back to `div`?

Reply:
I think I got confused between header and head

Reply:
I think semantically I prefer `header`. My understanding is that it's essentially just a named div.

Reply:
Have updated the other templates to match